jrcc-access-spring-boot-autoconfigure/src/test/java/ca/bc/gov/open/jrccaccess/autoconfigure/plugins/rabbitmq/RedisStorageServiceTests.java

```diff
@@ -116,5 +116,12 @@ public void getString_with_missing_document_should_throw_DocumentNotFoundExcepti
 		@SuppressWarnings("unused")
 		String expected = sut.getString(MISSING_DOCUMENT, "098A");
 	}
+
+        @Test
+	public void deleteString_with_valid_input_should_return_true() throws DocumentMessageException {
```

Comment:
Can you test the case where it throw the exception that we get a `DocumentMessageException`?